source/portable/NetworkInterface/xilinx_ultrascale/NetworkInterface.c

```diff
@@ -453,6 +473,29 @@ BaseType_t xGetPhyLinkStatus( void )
     return xReturn;
 }
 /*-----------------------------------------------------------*/
+NetworkInterface_t * pxFillInterfaceDescriptor( BaseType_t xEMACIndex,
+                                                NetworkInterface_t * pxInterface )
+{
+    static char pcName[ 17 ];
```

Comment:
As soon as multiple MAC's are supported, the `pcName[]` should host multiple names., like e.g.:
~~~
static char pcName[ XPAR_XEMACPS_NUM_INSTANCES ][ 17 ];
~~~
Or we could `malloc()` the space needed and pass it to `pxInterface->pcName`.